README.md

```diff
@@ -61,3 +64,13 @@ Create a cell in the top of the notebook and paste the content (example below)
 - Stephane Leonard for source-code and documentation improvements :)
 
 Contributions are welcomed!
+
+# Security
+
+For security issues please contact [security@getyourguide.com](mailto:security@getyourguide.com).
+
+# Legal
+
+Copyright 2021 GetYourGuide GmbH.
```

Comment:
This line makes it sound like it is closed-source. Wdyt?

Reply:
Does not sound like a big deal for GYG (as the project is already on gyg github org). I will just remove this line, if you feel strong about it we can add it back later.